t/engine.t

```diff
@@ -1159,8 +1171,8 @@ is_deeply $engine->seen, [
 ], 'Should have deployed to change 2';
 is_deeply +MockOutput->get_info, [
     [__x 'Deploying changes to {destination}', destination =>  $engine->destination ],
-    # [__ 'ok'],
```

Comment:
Him, why would these go away?

Reply:
I have no objection to leaving them but do you mind if I ask why they are there?

Reply:
Oh, I mis-read them as output (some output starts with `#`), but since they're commented out, yeah, fine to remove them.